hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/SCMContainerManager.java

```diff
@@ -217,7 +217,7 @@ public ContainerInfo getContainer(final ContainerID containerID)
       Collections.sort(containersIds);
 
       return containersIds.stream()
-          .filter(id -> id.getId() > startId)
+          .filter(id -> id.getId() >= startId)
```

Comment:
The interface doc says `startId` is exclusive:

https://github.com/apache/hadoop-ozone/blob/cd3f8c9f8316ae742ffeb497ab77581a4519173c/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManager.java#L77-L92
I don't know how "backward compatible" the interface needs to be at this stage, but I think it would be better to change the default `startId` to 0 in `scmcli` instead.  Otherwise, please update the javadoc, too.

CC @anuengineer

Reply:
Agree, +1 to change the startId to 0 for the scmcli list container command.

Reply:
+1 from me too. we can break backward compact now, no issues at all. We are still at the Alpha tag. As soon as we move to beta rules change :)

Reply:
If the user specifies startID =1 , it lists containers starting from container id#2. (not container id#1)
I think it would be good if the startID is kept intact and make the iteration of container inclusive of startID for better user experience.

Reply:
@anuengineer , @xiaoyuyao  -  Can you please take a look ?

Reply:
+1 on changing the `startId` to 0.
If we change the behavior of `listContainer` it will make it complex to implement a remote iterator to continuously list all the containers.